git_buildpackage

```diff
@@ -162,10 +172,32 @@ def create_source_package(repo_dir, output_dir, revision, build_args):
     # Since gbp is the "heart" of this service lets force interactive mode
     safe_run(command, cwd=repo_dir, interactive=True)
 
+    # Lets find the .dsc
+    #if not os.path.exists(
+    found_dsc = False
+    if append_git_id or append_git_url:
+        for fname in os.listdir(absoutput_dir):
+            if  fnmatch.fnmatch(fname, '*.dsc'):
+                input_file = os.path.join(absoutput_dir, fname)
+                found_dsc = True
+                with open(input_file, 'a') as dsc_file:
+                    if append_git_id:
+                        dsc_file.write('Git-Id: ' + git_id)
```

Comment:
Okay Daneil, I'll have a little look more at this, and come up with something:

 https://www.debian.org/doc/debian-policy/ch-controlfields.html

5.7 User-defined fields

Additional user-defined fields may be added to the source package control file. Such fields will be ignored, and not copied to (for example) binary or Debian source control files or upload control files.

If you wish to add additional unsupported fields to these output files you should use the mechanism described here.

Fields in the main source control information file with names starting X, followed by one or more of the letters BCS and a hyphen -, will be copied to the output files. Only the part of the field name after the hyphen will be used in the output file. Where the letter B is used the field will appear in binary package control files, where the letter S is used in Debian source control files and where C is used in upload control (.changes) files.

For example, if the main source information control file contains the field

       XBS-Comment: I stand between the candle and the star.
then the binary and Debian source control files will contain the field

       Comment: I stand between the candle and the star.